### DISCUSSION TITLE
Equation is not shown properly when using cross referencing and markdown output

### DISCUSSION THREAD

**Discussion by albertomercurio** [quarto-dev/quarto-cli#11617] [Q&A]
### Bug description

If I write an equation and I use cross-reference, it doesn't show properly on the markdown output file.

### Steps to reproduce

````qmd
---
engine: julia
---

$$
\hat{H}^{\mathrm{rwa}} = \sum_{j=0}^{N-1} \omega_j \vert \psi_j \rangle \langle \psi_j \vert + \omega_c \hat{a}^\dagger \hat{a} - i g \left( \hat{n}_t^+ \hat{a}^\dagger - \hat{n}_t^- \hat{a} \right) \, .
$$ {#eq-transmon-cavity-rwa}

Some text @eq-transmon-cavity-rwa.
````

with the `_quarto.yml` file

```yml
project:
  title: "Transmon Readout"

format:
  gfm:
    preview-mode: raw
    code-fold: true

execute:
  eval: false
  freeze: true
```

And the output file

````md


<span id="eq-transmon-cavity-rwa">$$
\hat{H}^{\mathrm{rwa}} = \sum_{j=0}^{N-1} \omega_j \vert \psi_j \rangle \langle \psi_j \vert + \omega_c \hat{a}^\dagger \hat{a} - i g \left( \hat{n}_t^+ \hat{a}^\dagger - \hat{n}_t^- \hat{a} \right) \, .
 \qquad(1)$$</span>

Some text
<a href="#eq-transmon-cavity-rwa" class="quarto-xref">Equation 1</a>.

````

Which is not displayed properly on both VSCode preview and also many online markdown previews. It works here (see the rendered text below). I was wondering if there exist another way to handle this.

### Rendered output (It works here)

<span id="eq-transmon-cavity-rwa">$$
\hat{H}^{\mathrm{rwa}} = \sum_{j=0}^{N-1} \omega_j \vert \psi_j \rangle \langle \psi_j \vert + \omega_c \hat{a}^\dagger \hat{a} - i g \left( \hat{n}_t^+ \hat{a}^\dagger - \hat{n}_t^- \hat{a} \right) \, .
 \qquad(1)$$</span>

Some text
<a href="#eq-transmon-cavity-rwa" class="quarto-xref">Equation 1</a>.

### Quarto check output

```bash
Quarto 1.6.32
[✓] Checking environment information...
      Quarto cache location: /home/alberto/.cache/quarto
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.4.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.46.3: OK
      Typst version 0.11.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.6.32
      Path: /opt/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: Installation From Path
      Path: /usr/bin
      Version: 2021

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.12.1 (Conda)
      Path: /home/alberto/anaconda3/envs/physics/bin/python
      Jupyter: 5.7.2
      Kernels: python3, julia-1.10, julia-1.11

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........(None)

      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/
```

**Comment by mcanouil** [comment]
Markdown does not support equation on its own. For equation to be displayed, a third party is required.

GFM stands for GitHub Flavoured Markdown.
GitHub uses MathJax to render equation. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions

I don't see what Quarto can do here.

**Comment by cscheid** [comment]
@mcanouil is correct. I'm moving this to a discussion so that it's recorded, but there's nothing Quarto itself can do about feature support in target formats.
